source/System.cpp

```diff
@@ -213,6 +215,10 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 			child.PrintTrace("Error: Expected key to have a value:");
 			continue;
 		}
+		else if(!remove && child.Size() == valueIndex + 1 && key == "linger time")
+			requestedLingerTime = child.Value(valueIndex);
+		else if(key == "linger time")
+			child.PrintTrace("Invalid \"linger time\" specification.");
```

Comment:
Now that you mention it we prob dont need this since the one at the end will pretty much say the same thing

Reply:
That would only save two lines, and it would make the message less informative.

Reply:
then I think you should give the proper specification or explain what is invalid
because honestly there isnt much difference between `Skipping unrecognized attribute: "linger time"` and this

Reply:
The full information would be too long. I removed it instead. Should someone ask me to put it back, I'll come up with something.